internal/asm/f64/ge_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build go1.7
+
+package f64
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gonum/blas/testblas"
+)
+
+func TestGer(t *testing.T) {
+	tests := []struct {
+		m, n            uintptr
+		alpha           float64
+		x, y, a         []float64
+		incX, incY, lda uintptr
+		want            []float64
+	}{
+		{
+			m: 1, n: 1, alpha: 1,
```

Comment:
The testing looks like unfinished. This should definitely be more thorough.

Reply:
I was using `testblas` for more thorough tests, but I'll copy those test cases over since using testblas requires making a copy of it elsewhere (import cycle).

Reply:
I think the tests here should test with respect to details such as the blocking and the block sizes. This is something that `testblas` does not need to test and should not be expected to test (although we have tests in this direction for Zaxpy).

Reply:
Testing incorporated.